pyfr/libs/ascent.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from ctypes import c_char_p, c_void_p, POINTER
+
+from pyfr.ctypesutil import LibWrapper
+
+
+class AscentWrappers(LibWrapper):
+    _libname = 'ascent_mpi'
+
+    # Functions
+    _functions = [
+        (None, 'ascent_close', c_void_p),
+        (c_void_p, 'ascent_create', c_void_p),
+        (None, 'ascent_execute', c_void_p, c_void_p),
+        (None, 'ascent_open', c_void_p, c_void_p),
+        (None, 'ascent_publish', c_void_p, c_void_p),
+    ]
+
+
+class Ascent:
```

Comment:
Probably no need for this class.  Just have the plugin use the low level wrappers like most other applications do.

Reply:
I think the current aaproach is cleaner and might be needed for future functionality.

Reply:
For Ascent?  The selling point is that it only has five function calls and they're all simple.